src/main/java/com/bobocode/blyznytsia/bibernate/exception/TransactionException.java

```diff
@@ -0,0 +1,25 @@
+package com.bobocode.blyznytsia.bibernate.exception;
+
+/**
+ * Indicates that a transaction could not be begun, committed or rolled back.
+ */
+public class TransactionException extends RuntimeException {
```

Comment:
Please, use `BibernateException` as a base instead of `RuntimeException`

Reply:
I think we have some misunderstanding here, `TransactionException` is totally fine. Just extend it from `BibernateException` instead of `RuntimeException`